index.html

```diff
@@ -81,11 +81,105 @@ <h1>Skills</h1>
                         <img src="IMG1/twitter.png" class="twitter" alt="">
                         <img src="IMG1/m.png" class="m" alt="">
                     </div>
-                    <div class="work-image">
-                        <img src="IMG1/work-element.png" alt="">
-                    </div>
                 </div>
         </section>
+
+        <section>
+            <div class="main-content-area">
+                <div class="work-image">
+                    <img src="IMG1/work-element.png" alt="">
+                </div>
+                <div class="title-container">
+                    <h1>MY RECENT WORK</h1>
+                    <img src="IMG1/divider-vector.png" alt="">
+                </div>
+                <div class="projects-container">
+                    <div class="image-placeholder">
+                        <img src="IMG1/placeholder.png" alt="">
+                    </div>
+                    <div class="content">
+                        <div class="background-img1">
+                        </div>
+                            <div class="text-area">
+                                <h1>Multi-Post Stories</h1>
+                                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                            </div>
+                        <ul class="list-tags">
+                            <li class="border-right">css</li>
+                            <li class="border-right">html</li>
+                            <li class="border-right">bootstrap</li>
+                            <li>Ruby</li>
+                        </ul> 
+                        <button type="button" class="myprojects">See projects</button>
+                    </div>
+                    
+
+                    <div class="projects-container">
+                        <div class="image-placeholder">
+                            <img src="IMG1/placeholder.png" alt="">
+                        </div>
+                        <div class="content">
+                            <div class="background-img2">
+                            </div>
+                                <div class="text-area">
+                                    <h1>Multi-Post Stories</h1>
```

Comment:
- [ ] Kindly change the h1 tag to h2, it is better for your project to have just one h1 tag, this helps screen readers read your website better 🤗